addons/sms/models/sms_api.py

```diff
@@ -56,3 +56,19 @@ def _send_sms_batch(self, messages):
             'messages': messages
         }
         return self._contact_iap('/iap/sms/2/send', params)
+
+    @api.model
+    def _get_sms_api_error_messages(self):
```

Comment:
Can't we return error_msg based on an error parameter ? An if-else would be great also to avoid adding dict each time.

Reply:
The thinking here is that if you test with 20 SMS numbers.
And each of those returns a "insufficient_credit" error (which will happen, if you have no credits left).
If I were to call the method 20 times with that error type, it would in turn create 20 calls to `self.sudo().env['iap.account'].get_credits_url(service_name='sms')`, which makes a RPC.

I wanted to avoid that, hence the dict I built in the sms testing code first (and then moved here as part of a previous suggestion).

I'm not sure I'm being very clear so ping me if needed on the communication tool of your choice ^^

Reply:
Indeed, did not see there was a call to iap.

Reply:
Maybe this is worth a small comment by the way.